### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by mwyatt896**
Added the following Puzzle contributions:

- Where's Alex
- Treasure Hunt
- Super Leo Ligue
- Aleo Fortune
- Puzzle SDK for zkGames
- Puzzle Wallet
- Puzzle SDK
- Aleo Wallet Connect Standards
- iOS Swift Starter Kit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,6 +160,11 @@ The following is a curated list of applications powered by Aleo.
 
 ### Gaming
 
+- [Where's Alex](https://github.com/puzzlehq/wheres_alex) simple multiparty hidden information game - a game of hide and seek with Alex Pruden! The most forked game on Aleo built using Puzzle SDK for zkGames.
```
Comment (collinc97) [inline README.md:163]:
```suggestion
- [Where's Alex](https://github.com/puzzlehq/wheres_alex) - A multiparty hidden information game on Aleo
```

File: README.md [hunk 2]
```diff
@@ -160,6 +160,11 @@ The following is a curated list of applications powered by Aleo.
...
+- [Treasure Hunt](https://treasures.puzzle.online/) multiparty hidden information game --built using Puzzle SDK for zkGames and Puzzle's game bot infrastructure
```
Comment (collinc97) [inline README.md:164]:
```suggestion
- [Treasure Hunt](https://treasures.puzzle.online/) - A multiparty hidden treasure hunt game on Aleo
```

File: README.md [hunk 3]
```diff
@@ -160,6 +160,11 @@ The following is a curated list of applications powered by Aleo.
...
+- [Super Leo Ligue](https://aleo-fortune.vercel.app/) Play classic roulette and more onchain -- built using Puzzle SDK for zkGames
```
Comment (collinc97) [inline README.md:165]:
```suggestion
- [Super Leo Ligue](https://aleo-fortune.vercel.app/) - Play classic casino games like roulette and more on Aleo
```

File: README.md [hunk 4]
```diff
@@ -160,6 +160,11 @@ The following is a curated list of applications powered by Aleo.
...
+- [Aleo Fortune](https://superleolig.vercel.app/) challenge a friend to an onchain soccer game with hidden strategies -- built using Puzzle SDK for zkGames
```
Comment (collinc97) [inline README.md:166]:
```suggestion
- [Aleo Fortune](https://superleolig.vercel.app/) - Challenge a friend to an onchain soccer game with hidden strategies
```

File: README.md [hunk 5]
```diff
@@ -160,6 +160,11 @@ The following is a curated list of applications powered by Aleo.
...
+- [Puzzle SDK for zkGames](https://docs.puzzle.online/tutorials/build_your_first_zkgame/) purpose built SDK for building multiparty hidden information games on Aleo by the Puzzle Wallet team
```
Comment (collinc97) [inline README.md:167]:
```suggestion
- [Puzzle SDK for zkGames](https://docs.puzzle.online/tutorials/build_your_first_zkgame/) - A SDK for building multiparty hidden information games on Aleo
```

**Review by collinc97 — CHANGES_REQUESTED** [review]
Few nits. Thank you Puzzle team!

**Review by howardwu — APPROVED** [review]

**Pull request merged**